apps/src/utils.js

```diff
@@ -394,6 +405,19 @@ export function trySetLocalStorage(item, value) {
 
 }
 
+export function tryGetSessionStorage(key, defaultValue) {
+  if (defaultValue === undefined) {
+    throw "tryGetSessionStorage requires defaultValue";
+  }
+  let returnValue = defaultValue;
+  try {
+    returnValue = sessionStorage.getItem(key);
```

Comment:
I would expect this function to return `defaultValue` if there is no item for `key`, but this just returns `null`, right? I don't think it throws an exception for missing keys.

Reply:
correct, and I agree that it's a bit weird but I wanted to follow the pattern of [`tryGetLocalStorage`](https://github.com/code-dot-org/code-dot-org/pull/17220/files#diff-85c58ca2effd283f2dfd4479a8777179R378) which is operating the same way